src/components/common/PeripheralsDialog.vue

```diff
@@ -368,7 +368,8 @@ export default class ManualProbeDialog extends Mixins(StateMixin) {
         type: 'can',
         count: this.canbusUuids
           ? Object.values(this.canbusUuids)
-            .reduce((a, b) => a + b.length, 0)
+            .filter(value => Array.isArray(value))
```

Comment:
Not sure I understand the need for this check, according to Moonraker documentation, this is always an array (empty or not), never null or anything else!